src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  sourceString
+    .trim()
+    .split(';')
+    .forEach((element) => {
+      const [key, value] = element.split(':');
+
+      if (value !== undefined) {
+        obj[key.trim()] = value.trim().replace(/\\n/g, '\n');
```

Comment:
Replacing '\n' with '
' is unnecessary here as there shouldn't be any newline characters inside the value after trimming. If you need to handle special cases, make sure they are expected in the task requirements.